azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/extensions/AzureComputeSecurityGroupExtension.java

```diff
@@ -135,12 +137,12 @@ public boolean apply(SecurityGroup input) {
       if (vm == null) {
          throw new IllegalArgumentException("Node " + nodeId + " was not found");
       }
-      List<IdReference> networkInterfacesIdReferences = vm.properties().networkProfile().networkInterfaces();
+      List<NetworkInterface> networkInterfacesIdReferences = vm.properties().networkProfile().networkInterfaces();
```

Comment:
[minor] is maybe `networkInterfaces` rather than `networkInterfacesIdReferences` ?